lib/velum/dex/oidc.rb

```diff
@@ -0,0 +1,24 @@
+require "base64"
+
+module Velum
+  # This class offers the integration between ruby and the Saltstack API.
```

Comment:
This comment doesn't make sense and/or doesn't seem to belong. Copied from elsewhere?

Reply:
I'll point at the `git blame` output to explain that one. ;)

This is the file which generates the list of OIDC connectors for SaltStack to consume.  Slightly more clear comment moved to top of file in fb2d6f5.